test/functional/long/typing-stress-e2e-specs.js

```diff
@@ -20,25 +23,24 @@ describe('XCUITestDriver - long tests', function () {
   });
 
   describe('typing', function () {
-    beforeEach(async function () {
+    let text = 'bunchoftext';
+
+    before(async function () {
       let el = await driver.elementByAccessibilityId('Text Fields');
       await driver.execute('mobile: scroll', {element: el, toVisible: true});
       await el.click();
     });
-    afterEach(async () => {
-      await driver.back();
-    });
 
-    it('should not fail in typing', async () => {
-      let text = 'bunchoftext';
-      let el = await driver.elementByClassName('XCUIElementTypeTextField');
-      for (let i = 0; i < 200; i++) {
+    for (let i = 0; i < TYPING_TRIES; i++) {
+      it(`should not fail in typing (try #${i+1})`, async () => {
+        let el = await driver.elementByClassName('XCUIElementTypeTextField');
```

Comment:
Yeah. I was trying to be fancy, to get the `it` inside the loop. The way mocha works makes it difficult to have the element stuff outside.